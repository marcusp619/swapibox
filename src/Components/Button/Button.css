/*------------------------------------*\
  #BUTTON
\*------------------------------------*/
.Button {
  background: black;
  border: none;
  color: white;
  width: 150px;
  height: 30px;
}
