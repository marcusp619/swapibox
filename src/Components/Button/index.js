import React, { Component } from 'react';
import './Button.css';

class Button extends Component {
  constructor(props) {
    super(props)
  }

  render() {
    return(
      <div className="Button">
        <h2>Button Component</h2>
      </div>
    )
  }
}

export default Button;