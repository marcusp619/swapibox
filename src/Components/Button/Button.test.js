import React from 'react';
import ReactDOM from 'react-dom';
import Button from './'
import { shallow } from 'enzyme';

describe('Button', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<Button />);

    expect(wrapper).toMatchSnapshot();
  });
})
  