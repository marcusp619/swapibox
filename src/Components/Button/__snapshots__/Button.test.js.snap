// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders without crashing 1`] = `
<div
  className="Button"
>
  <h2>
    Button Component
  </h2>
</div>
`;
